contribs/gmf/src/services/editfeature.js

```diff
@@ -52,6 +68,56 @@ gmf.EditFeature.prototype.getFeatures = function(layerIds, extent) {
 
 
 /**
+ * Build a query to the MapFish protocol to fetch features from a list
+ * of layer ids and a list of comparison filters.
+ *
+ * This method is called in the ObjectEditing service, which is injected in
+ * the permalink service, i.e. it's always called. Since we don't have to
+ * define the url to the GMF Protocol (layers) a dummy promise returns an
+ * empty array of features if the url is not defined.
+ *
+ * @param {Array.<number>} layerIds List of layer ids to get the features from.
+ * @param {Array.<gmfx.ComparisonFilter>} filters List of comparison filters
+ * @return {angular.$q.Promise} Promise.
+ * @export
+ */
+gmf.EditFeature.prototype.getFeaturesWithComparisonFilters = function(
```

Comment:
Should this method be accessed from a partial? If not it should not be marked `@export`.